test/modules/standard/List/memory.chpl

```diff
@@ -0,0 +1,19 @@
+use Memory;
+use List;
+
+var mem : uint;
+proc output()
+{
+	mem=memoryUsed();
```

Comment:
Oh, I forgot to mention earlier.  This test needs to be run with --memTrack, right?  If so, you should add a memory.execopts file specifying that:

````
--memTrack
````

Otherwise, I think running util/start_test on this test will fail?